dict

```diff
@@ -19,9 +20,11 @@ else:
                 if (u'class', u'collins_content') in x.attrs] #try the Collins Dict
         if not dict: #if no match in Collins, try the machine translation
             dict = [x for x in soup.findAll('div')\
-                    if (u'class', u'group_pos') in x.attrs]
+                    if (u'class', u'info-article info-base') in x.attrs]
         result = dict[0].getText(' ') #only show the first match, which seems best
         result = re.sub('&nbsp;', ' ', result) #replace the HTML space symbol
         result = re.sub('[\s]{4,}', '\n', result) #remove redundent spaces
         result = re.sub('\d+\. ', add_new_line, result) #highlight the groups
+        result = re.sub("\(function\(\)[\s\S]*",' ', result)
+        result = re.sub(u"一元好课[\s\S]*",' ', result)
```

Comment:
try using a wilder pattern so that if the website changes again we don't need to change the code.